test/se/vidstige/jadb/AdbInputStreamFixtureTest.java

```diff
@@ -1,16 +1,14 @@
-package se.vidstige.jadb.test.unit;
+package se.vidstige.jadb;
 
 import org.junit.Assert;
 import org.junit.Test;
-import se.vidstige.jadb.AdbFilterInputStream;
-import se.vidstige.jadb.Stream;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
-public class AdbInputStreamFixture {
+public class AdbInputStreamFixtureTest {
```

Comment:
Only possible solution to separate  `unit` tests from `integration` test is to give them different names....
It is ugly but only possible solution - I have investigated this problem for 2 hours and did not found a solution. 

P.S. Nevertheless we can choose another name pattern, here are [defaults](http://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html)